packages/app/src/vis-packs/core/raw/MappedRawVis.tsx

```diff
@@ -0,0 +1,57 @@
+import { RawImageVis, RawVis } from '@h5web/lib';
+import {
+  type ArrayShape,
+  type Dataset,
+  type ScalarShape,
+} from '@h5web/shared/hdf5-models';
+import { createPortal } from 'react-dom';
+
+import visualizerStyles from '../../../visualizer/Visualizer.module.css';
+import { useExportEntries } from '../hooks';
+import { type RawConfig } from './config';
+import RawToolbar from './RawToolbar';
+import { isBinaryImage } from './utils';
+
+interface Props {
+  dataset: Dataset<ScalarShape | ArrayShape>;
+  value: unknown;
+  toolbarContainer?: HTMLDivElement | undefined;
+  config: RawConfig;
+}
+
+function MappedRawVis(props: Props) {
```

Comment:
I had to create a component to be able to call `useExportEntries`. There's no dimension mapper in the raw vis so it's a bit counter-intuitive, but I may need to add one soon to support stacks of binary images.